src/app/container/info-tab/info-tab.component.css

```diff
@@ -13,3 +13,8 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+
+.mat-icon {
```

Comment:
https://stackoverflow.com/a/56958446/4484759

Reply:
Thanks, just added this!